articles/quickstart/native/android/06-calling-apis.md

```diff
@@ -22,23 +25,53 @@ This tutorial shows you how to use a previously saved token to authenticate your
 
 Before you continue with this tutorial, make sure that you have completed the previous tutorials. This tutorial assumes that:
 * You have completed the [Session Handling](/quickstart/native/android/03-session-handling) tutorial and you know how to handle the `Credentials` object.
-* You have set up a backend application as API. To learn how to do it, follow one of the [backend tutorials](https://auth0.com/docs/quickstart/backend). 
+* You have set up a backend application as API. To learn how to do it, follow one of the [backend tutorials](https://auth0.com/docs/quickstart/backend).
+
 
-After you set up an API, declare the endpoint you need to call as a constant in the current class:
+<%= include('_includes/_calling_api_create_api') %>
+
+<%= include('_includes/_calling_api_create_scope') %>__
```

Comment:
stray underscores after your include?

Reply:
With underscores
![image](https://user-images.githubusercontent.com/3900123/32065932-7d7d365e-ba54-11e7-806e-d541b9a911fd.png)

Without underscores
![image](https://user-images.githubusercontent.com/3900123/32065943-86c8cff2-ba54-11e7-964b-c5b7713e9238.png)

I know then don't belong there, but they don't hurt also and help syntax parsers like the Atom's.

Reply:
Haha, I might have to start doing that myself then. As long as it won't bother anything when it builds it doesn't bother me.

Reply:
It shouldn't as it's only "opening italics" and "closing italics". You can a space in the middle, or put it in the line below. All are fine 😝

Reply:
@lbalmaceda These links are wrong, this links to to the _include folder in the Android directory. should be `../_includes`